### PR TITLE
docs(admin) add docs for vault_beta entity

### PR DESCRIPTION
**Pull request by bungle**
### Summary

Adds "autogen" docs to vault_beta entity.

### CODE REVIEW

File: autodoc/admin-api/data/admin-api.lua [hunk 1]
```diff
@@ -1885,7 +1886,66 @@ return {
...
+    vaults_beta = {
+      title = "Vaults Beta Entity",
+      entity_title = "Vault",
+      entity_title_plural = "Vaults",
+      description = [[
+        Vault entities are used to configure different Vault connectors. Examples of
+        Vaults are Environment Variables, Hashicorp Vault and AWS Secrets Manager.
+
+        Configuring a Vault allows referencing the secrets with other entities. For
+        example a certificate entity can store a reference to certificate and key,
```
Comment (jschmid1) [inline autodoc/admin-api/data/admin-api.lua:1900]:
```suggestion
        example a certificate entity can store a reference to a certificate and key,
```

File: autodoc/admin-api/data/admin-api.lua [hunk 2]
```diff
@@ -1885,7 +1886,66 @@ return {
...
+          description = [[
+            The name of the Vault that's going to be added. Currently, the Vault implementation
+            must be installed in every Kong instance.
+          ]],
+          example = "env",
+        },
+        prefix = {
+          description = [[
+            The unique prefix (or identifier) for this Vault configuration. The prefix
+            is used to load right Vault configuration and implementation when referencing
```
Comment (jschmid1) [inline autodoc/admin-api/data/admin-api.lua:1920]:
```suggestion
            is used to load the right Vault configuration and implementation when referencing
```

**Review by jschmid1 — COMMENTED** [review]
see suggestions

**Comment by bungle** [comment]
> see suggestions

Thanks, committed them!

**Review by jschmid1 — APPROVED** [review]

**Pull request merged**